model/tab/gear/weapon-main.html

```diff
@@ -53,7 +53,6 @@
         <input name="data.features" type="text" value="{{data.features}}" />
     </div>
 </div>
-
 <div class="roll-modifiers border">
-{{> systems/forbidden-lands/model/partial/roll-modifiers.html }}
-</div>
\ No newline at end of file
+    {{> systems/forbidden-lands/model/partial/roll-modifiers.html }}
```

Comment:
Silly question : the tabulation before this partial inclusion isn't an issue because `roll-modifiers.html` does not contain any _text input_, right?

Reply:
It's an ooops. Technically should be fine, but yeah, auto format on save happened, and I didn't fix it.